src/backend/static/styles/style.css

```diff
@@ -0,0 +1,74 @@
+/* General */
+
+html {
+  /*font-family: 'Raleway', sans-serif;*/
+  background-color: rgb(219, 217, 217);
+  font-size: 10px;
+}
+
+body {
+  width: 800px;
+  margin: 0 auto;
+}
+
+/* Typography */
+
+h1 {
+  /*font-family: 'Cherry Swash', cursive;*/
+  margin: 0;
+  text-align: center;
+  font-size: 5rem;
+  background-size: 48px 48px;
+}
+
+input, p {
+  font-size: 1.5rem;
+}
+
+/* block styling */
+
+header, left, right, footer {
+  padding: 20px;
+  border: 2px solid rgb(100, 98, 98);
+  border-bottom: 0;
+}
+
+body > div {
+  border: 2px solid rgb(100, 98, 98);
+  display: flex;
+}
+
+left {
+  border-right: 2px solid rgb(100, 98, 98);
+  flex: 5;
+  display: flex;
+  flex-flow: column;
+  align-items: stretch;
+}
+
+right {
+  display: flex;
+  flex: 5;
+  flex-flow: column;
+  align-items: stretch;
+}
+
+left div {
+  margin-bottom: 20px;
+}
+
+input {
+  display: block;
+  background: rgb(71, 70, 70);
+  box-shadow: inset 1px 1px 1px white;
+  color: rgb(0, 0, 0);
+  border: none;
```

Comment:
Properties with irrelevant values should be omitted. I'm not 100% sure but I think this applies to this case.